packages/opentelemetry-api/typedoc.tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Just curious to know, why we are splitting `typedocOptions` in a separate file?

Reply:
it's an attempt to fix the broken build. I really don't know what is going on